ci/scripts/sql/nexmark/q19.sql

```diff
@@ -0,0 +1,12 @@
+-- noinspection SqlNoDataSourceInspectionForFile
+-- noinspection SqlResolveForFile
+CREATE SINK nexmark_q19 AS
+SELECT auction, bidder, price, channel, url, date_time, extra 
```

Comment:
trailing white space🧐

Reply:
Extracted directly from kube-bench via https://github.com/risingwavelabs/kube-bench/blob/main/scripts/extract-queries.sh, so won't fix in this PR. Need to patch it in kube-bench repo.